src/Microsoft.TestPlatform.CommunicationUtilities/ProtocolVersioning.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.ObjectModel;
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+internal static class ProtocolVersioning
```

Comment:
Would it be worth to replace this static class with an enum?

Reply:
Not yet. I am not sure what exactly this class will do. Right now it is just holding numbers, but maybe it won't in the future.